easybuild/easyblocks/n/namd.py

```diff
@@ -67,6 +57,23 @@ def __init__(self, *args, **kwargs):
         super(EB_NAMD, self).__init__(*args, **kwargs)
         self.namd_arch = None
 
+    def prepare_step(self, *args, **kwargs):
+        """Prepare build environment."""
+        super(EB_NAMD, self).prepare_step(*args, **kwargs)
+
+        if self.cfg['namd_basearch'] is None:
+
+            self.log.info("namd_basearch not specified, so determining it based a CPU arch & CUDA dep...")
```

Comment:
I guess mentioning the CUDA dep here is misleading then

Reply:
fixed in 32b61e8